.ci/azure/android_arm64.yml

```diff
@@ -62,8 +62,23 @@ jobs:
     TMP_DIR: /mnt/tmp
     SHARE_DIR: /mount/cinfsshare/onnxtestdata
     CCACHE_DIR: $(SHARE_DIR)/ccache/master/android_arm64
+    LD_LIBRARY_PATH: $(Agent.ToolsDirectory)/Python/$(OV_PYTHON_VERSION)/x64/lib
+    OV_PYTHON_VERSION: 3.10.10 # Newest version of Python 3.10. Full version of Python its required for LD_LIBRARY_PATH and required for successful Task setupPython
```

Comment:
I think we will have the same problem once new Python is release. Can we design it in more flexible way that will not require regular update?

Reply:
One of solution its using public images of VM for Microsoft-hosted agent like Ubuntu 22.04, Ubuntu 20.04. Those images include more versions of Python and they are updated by Microsoft frequently. That is why they not required variables like LD_LIBRARY_PATH because inside VM its already set all necessary path to tool cache. So only will be required task UsePython with specVersion: 3.9, 3.10, 3.11 etc. Unfortunately we using custom VM and it only have Python 3.8.10 what I heard on this image there are some tweaks that Konstantin Satunin and Alexander Zhogov did to speed up/improve Azure build a little bit, one example is installation of IncrediBuild.